en/index.json

```diff
@@ -112,5 +112,12 @@
     "electric_cost": "Electricity cost for 1 kWh",
     "estimate_again": "Estimate again",
     "use_detected_wattage" : "Use wattage data obtained from Miner"
+  },
+  "pay_me_now": {
+    "pay_me_now": "Pay Me Now!",
+    "payment_queued": "Payment queued.",
+    "payment_verification": "Payment Verification",
+    "payment_verification_explain": "To verify if your account can receive payments from Freedom!, we will pay this account an amount of $0.01",
+    "verify": "VERIFY"
```

Comment:
Lower case?

Reply:
```
“pay_me_now”：“馬上付錢！”，
    “payment_queued”：“已申請支付”，
    “payment_verification”：“付款驗證”，
    “payment_verification_explain”：“要驗證您的帳戶是否可以接受Freedom！的付款，我們將向此帳戶支付0.01美元”，
    “verify”：“驗證”,
    “verified”：“已驗證”
```

Is verify showing already verified badge? If so, the text should be "Verified" instead?

Reply:
![image](https://user-images.githubusercontent.com/4374443/44897958-ac3feb00-ad27-11e8-95d8-72e612e32c79.png)
it is for caption of button. I'm not so sure uppercase is different in other language or not (like Chinese) so I put it uppercase. For now, I guess we can do upper case in code.

Reply:
I see, in that case I think the original looks fine.

Reply:
I added Chinese translation.